src/lib.rs

```diff
@@ -799,7 +821,7 @@ where
         G: GrantType,
         JK: JweKeyManagementAlgorithm,
         RM: ResponseMode,
-        RT: ResponseType,
+        RT2: ResponseType,
```

Comment:
nit: how about renaming to `RS` instead of `RT2`

Reply:
Done.